### ISSUE TITLE
Duplicate endpoint at module upgrade

### ISSUE THREAD

**Issue by drkloc** [NuCivic/dkan#416]
_From @upieper on September 29, 2014 19:58_

I had forked odsm at revision 46434afc6081699e342ece5b1cd932da1c12a243. 
After rebasing to HEAD on Friday, the ckan_revision_list endpoint now appears twice in `admin/config/services/odsm`. 

Output of `$list` in `function open_data_schema_map_api_load_all()`: 

```
array (
  0 => 'ckan_group_list',
  1 => 'ckan_package_list',
  2 => 'ckan_package_show',
  3 => 'ckan_revision_list',
  4 => 'data_json',
  5 => 'out_own_endpoint',
  6 => 'ckan_site_read',
  7 => 'ckan_revision_list',
  8 => 'ckan_package_revision_list',
)
```

The first entry comes from a database entry that was created in a previous version. 
The second entry is generated from code. 

Is the solution to this issue, to remove the endpoint from the database table? 
Should that have been done in the .install script? 

_Copied from original issue: NuCivic/open_data_schema_map#19_

**Comment by drkloc** [comment]
_From @upieper on September 30, 2014 14:27_

Here is a bit more information. 

I have adjusted my open_data_schema_iad module to mimic the new version of open_data_schema_ckan. Additionally, I have created a feature module that includes the mapping. 
After I disable these two, both entries to the iad api are removed from ```admin/config/services/odsm```. 

When I enable them, two entries are again listed:
![screen shot 2014-09-30 at 10 22 54](https://cloud.githubusercontent.com/assets/2594179/4459727/a7733d9e-48ad-11e4-91b7-0be5aff2f8eb.png)

Editing either of them updates the database entry. This functions as expected, but appears odd.
